### PR TITLE
Fixes #19646: Secrets variables event log

### PR DESCRIPTION
**Pull request by ElaadF**
https://issues.rudder.io/issues/19646
We should merge https://github.com/Normation/rudder/pull/3793 first before merging this

### CODE REVIEW

**Comment by ElaadF** [comment]
PR updated with a new commit

File: webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/marshalling/XmlUnserialisationImpl.scala [hunk 1]
```diff
@@ -760,3 +761,21 @@ class ApiAccountUnserialisationImpl extends ApiAccountUnserialisation {
...
+  def unserialise(entry:XNode) : Box[Secret] = {
+    for {
+      secret      <- {
+        if(entry.label ==  XML_TAG_SECRET) Full(entry)
+        else Failure("Entry type is not a <%s>: %s".format(XML_TAG_SECRET, entry))
+      }
+      fileFormatOk     <- TestFileFormat(secret)
+
+      name             <- (secret \ "name").headOption.map( _.text ) ?~! ("Missing attribute 'name' in entry type secret : " + entry)
+      //      value            <- (secret \ "value").headOption.map( _.text ) ?~! ("Missing attribute 'value' in entry type secret : " + entry)
```
Comment (ElaadF) [inline webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/marshalling/XmlUnserialisationImpl.scala:775]: to remove

**Review by VinceMacBuche — APPROVED** [review]

**Comment by ElaadF** [comment]
PR updated with a new commit

**Comment by ElaadF** [comment]
PR updated with a new commit

**Review by VinceMacBuche — APPROVED** [review]

**Pull request merged**